src/Common/src/System/Security/Cryptography/Asn1V2.cs

```diff
@@ -0,0 +1,5246 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Buffers;
+using System.Collections.Generic;
+using System.Diagnostics;
+using System.Numerics;
+using System.Runtime.InteropServices;
+using System.Text;
+
+namespace System.Security.Cryptography.Asn1
```

Comment:
This file is large enough to cause tooling problems. Consider splitting into multiple files.

Reply:
Split.  All the comments will switch to "old code" when I push the commit momentarily.